src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -94,6 +94,13 @@ public function initOrm()
 
         // name for second category
         $categoryTrans2 = new CategoryTranslation();
+        $categoryTrans2->setLocale('en');
+        $categoryTrans2->setTranslation('Second Category');
+        $categoryTrans2->setCategory($category2);
+        $category2->addTranslation($categoryTrans2);
```

Comment:
Did you have to add a second translation? Actually this test should also work with only one language.